README.md

```diff
@@ -1 +1,85 @@
-# api-info-movie-clean-code-tdd
\ No newline at end of file
+# :rocket: Info Movie API
+
+This api is able to search for films from a public database and find its translations and save it in a database in its own docker container, and was developed by Esdras Aguilar using TDD and Clean Architecture.
+
+## :vertical_traffic_light: Getting Started
+
+Follow this instructions to run the project
+
+### :minidisc: Installing
+
+We recommend using Yarn, but you can use npm
+
+  **Locally**
+ 
+  * You need to install the dependencies: 
+  
+    **Using NPM:**
+    ```
+    npm install
+    ```
+    **Using Yarn:**
+    ```
+    yarn install
+    ```
+    * Rename the file ".env.default" to ".env" and setup enviroment variables to connect to MongoDb and config port server.
+    
+  * Start the application:
+  
+    **Using NPM:**
+    ```
+    npm start
+    ```
+    **Using Yarn:**
+    ```
+    yarn start
+    ```
+
+  **Docker**
+
+   * First you need to have the [Docker](https://hub.docker.com/_/node/) and [Docker-Compose](https://docs.docker.com/compose/install/) installed on your computer;
+
+   * Rename the file ".env.default" to ".env" and setup enviroment variables to connect to MongoDb and config port server. PS.: If you can not connect in mongodb, run this CMD to find docker IP and put in MONGO_HOST variable:
+    ```
+    docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' db
+    ```
+
+    *And finally:
```

Comment:
The indentation is incorrect, if your intention is to create a topic, you should put it in the same spacing as the previous topic